### PR TITLE
fix(editable-layers): Make the implicit dependency to layers module explicit. #112

### LINKED ISSUE #112: Module editable-layers has implicit dependency to layers
The module editable-layers has no explicit dependency to module layers: https://github.com/visgl/deck.gl-community/blob/master/modules/editable-layers/package.json

In fact the segments-layer is importing the layers module: https://github.com/visgl/deck.gl-community/blob/646f8328d27d67d596f05c9154072051ec5cf4f0/modules/editable-layers/src/lib/layers/segments-layer.ts#L9

This causes a build failure for any project, that only imports `@deck.gl-community/editable-layers`, but not `@deck.gl-community/layers`. The workaround is to also import `@deck.gl-community/layers`, but this implicit dependency should be made explicit by defining layers as peerDepdendency for editable-layers.

### PR DESCRIPTION
**Pull request by cg-sat**
closes #112

### CODE REVIEW

**Comment by cg-sat** [comment]
@ibgreen Here's the first PR.

File: modules/editable-layers/package.json [hunk 1]
```diff
@@ -70,6 +70,7 @@
     "viewport-mercator-project": ">=6.2.3"
   },
   "peerDependencies": {
+    "@deck.gl-community/layers": "^9.0.2",
```
Comment (felixpalmer) [inline modules/editable-layers/package.json]: In deck.gl we have the peerDependencies defined more loosely, e.g. `"^9.0.0"`, any reason not to do that in this repo also?

Comment (cg-sat) [reply modules/editable-layers/package.json]: @felixpalmer Thanks for the help. I relaxed the version constraint according to your suggestion.-

**Review by ibgreen — APPROVED** [review]

**Comment by cg-sat** [comment]
@ibgreen I figured out, what was causing yarn to use version 1 behavior. By running `yarn set version stable` it writes a `packageManager` key into the root package.json. If this key isn't present, yarn seems to fall back to version 1 behavior. I added the change in the root package.json in my other MR #112 With that change I only get the necessary changes in the lock file.

**Pull request merged**